### PR TITLE
Code Review Project

### PR DESCRIPTION
**Pull request by Devtiwo**
In this project, I 

- Opened a new repository
- copied index and style files from the Admission trails process into the new repository so as to review them.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+            <li><a class = "nav-link" href = "#Variables">Variables</a></li>
+            <li><a class = "nav-link" href = "#Declaring_Variables">Declaring variables</a></li>
+            <li><a class = "nav-link" href = "#Variable_scope">Variable scope</a></li>
+            <li><a class = "nav-link" href = "#Global_variables">Global variables</a></li>
+            <li><a class = "nav-link" href = "#Constants">Constants</a></li>
+            <li><a class = "nav-link" href = "#Data_types">Data types</a></li>
+            <li><a class = "nav-link" href = "#If...else_statement">If...else statement</a></li>
+            <li><a class = "nav-link" href = "#While_statement">While statement</a></li>
+            <li><a class = "nav-link" href = "#Function_declarations">Function declarations</a></li>
+            <li><a class = "nav-link" href = "#Reference">Reference</a></li>
```
Comment (Devtiwo) [inline index.html:27]: Please make sure not to leave spaces when defining class for HTML elements. kindly correct them in your HTML tags where class or id is defined. They are best practices for HTML. You can read [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: style.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+body{
```
Comment (Devtiwo) [inline style.css:3]: Please make sure to leave a space after the selector before the opening curly brackets. Kindly check for the rest of the CSS selectors. They are CSS formatting guidelines.

**Review by Devtiwo — COMMENTED** [review]
Kindly check my comments below for required changes.

- Also, no linters were used. Kindly make sure to use linters to check for bugs and errors.
- No Descriptive README file. Kindly make sure to document your project so contributors can have a better understanding of what your project is about.
- please do make sure to add and commit your files individually during your next project.
 
Thanks :relieved:

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @Devtiwo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] Friendly code review, checked for both code improvements and making the repo more professional.

## Required Changes ♻️
- [ ] Great job pointing out those issues. According to the requirements, for all comments, you should explain why you are requesting those changes. For instance, you should explain why you should not leave spaces when defining class for HTML, you can even attach some documentation, like [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/class).

## Optional suggestions
- [ ] You could consider adding some emojis and gifs to make the code review more human-friendly 👽 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andersonlebon — APPROVED** [review]
Hi @Devtiwo 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub flow :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions 

- [ ] For better communication I think it would be nice to have a short INTRODUCTION to your review that can congratulate the student on their efforts in building the project and help him know which of those requirements are optional or required.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**